lib/puppet/file_serving/configuration/parser.rb

```diff
@@ -79,14 +85,20 @@ def deny(mount, value)
         mount.info _("denying %{val} access") % { val: val }
         mount.deny(val)
       rescue Puppet::AuthStoreError => detail
-        raise ArgumentError.new(_("%{detail} in %{file}, line %{line_num}") % { detail: detail.to_s, file: @file, line_num: @count })
+        error_location_str = Puppet::Util::Errors.error_location(@file, @count)
+        raise ArgumentError.new("%{detail} %{error_location}" %
```

Comment:
Should this have an " at "?  Inconsistent with line 99 below.

Reply:
Actually, specifically no, in this case. Since we don't know the error that occurred, we have not idea what the sentence looks like in this case so we just add the error_location which looks like this '(file: <path>, line: <line>)' since those are nouns, they are translatable without the context.